src/rpc/net.cpp

```diff
@@ -509,6 +511,395 @@ static RPCHelpMan getaddednodeinfo()
+                        /*inner=*/{
+                            {
+                                /*type=*/RPCResult::Type::OBJ_DYN,
+                                /*m_key_name=*/"keys for the second aggregate type",
+                                /*description=*/"",
+                                /*inner=*/{
+                                    {
+                                        /*type=*/RPCResult::Type::OBJ_DYN,
+                                        /*m_key_name=*/"keys for the third aggregate type",
+                                        /*description=*/"",
+                                        /*inner=*/net_stats::NetMsgStat()
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            // A return object with four levels of data: four aggregation types
+            RPCResult {
+                /*type=*/RPCResult::Type::OBJ_DYN,
+                /*m_key_name=*/"",
+                /*description=*/"",
+                /*inner=*/{
+                    {
+                        /*type=*/RPCResult::Type::OBJ_DYN,
+                        /*m_key_name=*/"keys for the first aggregate type",
+                        /*description=*/"Network traffic statistics.\n"
+                            "When a message, connection, network type, or direction is not listed,\n"
+                            "the statistics for that type are 0. Only known message,\n"
+                            "connection, and network types can appear as keys in the object.",
+                        /*inner=*/{
+                            {
+                                /*type=*/RPCResult::Type::OBJ_DYN,
+                                /*m_key_name=*/"keys for the second aggregate type",
+                                /*description=*/"",
+                                /*inner=*/{
+                                    {
+                                        /*type=*/RPCResult::Type::OBJ_DYN,
+                                        /*m_key_name=*/"keys for the third aggregate type",
+                                        /*description=*/"",
+                                        /*inner=*/{
+                                            {
+                                                /*type=*/RPCResult::Type::OBJ_DYN,
+                                                /*m_key_name=*/"keys for the fourth aggregate type",
+                                                /*description*/"",
+                                                /*inner*/net_stats::NetMsgStat()
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        /*examples=*/
+        RPCExamples {
```

Comment:
I think I just discovered an issue where `getnetmsgstats -h` doesn't display the help info. Leaving a note to self.

Reply:
In `src/rpc/client.cpp` I was able to find other RPCs that accepted arrays and they also display an error when passing in the `-h` flag.

```
error: Error parsing JSON: --h
```

As this behavior does not appear to be isolated to this PR, I am closing this comment.